xen/common/coverage/llvm.c

```diff
@@ -107,11 +145,34 @@ static int cf_check dump(
     struct llvm_profile_header header = {
         .magic = LLVM_PROFILE_MAGIC,
         .version = LLVM_PROFILE_VERSION,
-        .data_size = (END_DATA - START_DATA) / sizeof(struct llvm_profile_data),
-        .counters_size = (END_COUNTERS - START_COUNTERS) / sizeof(uint64_t),
+#if __clang_major__ >= 13
+        .binary_ids_size = 0,
+#endif
+        .num_data = ((END_DATA + sizeof(struct llvm_profile_data) - 1)
```

Comment:
Darko:
Maybe add a comment "// These roundings copied from <link_to_LLVM>" ??

Reply:
They don't have issue with rounding up. Maybe it would be better to not put this comment here to keep changes minimized